frontend/src/components/Utils/utils.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Format a timestamp (in milliseconds) into a pretty string with just the time.
+ * @param {int} msTimestamp 
+ * @param {string} timezone 
+ * @returns {string} Time formatted into a string like "10:19 AM".
+ */
+export function timestampToTimeFormatted(msTimestamp, timezone = "America/New_York") {
```

Comment:
Single quotes. Check this and other files in this PR because I see it a few times but don't want to spam the PR with these comments.
```suggestion
export function timestampToTimeFormatted(msTimestamp, timezone = 'America/New_York') {
```